dangerfile.js

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable */
```

Comment:
Because I was being lazy and I don't like the mithril repo eslint config 😉

I'll make sure it passes before submitting.